plugins/kubernetes/app/models/kubernetes/resource.rb

```diff
@@ -7,9 +7,10 @@ module Kubernetes
   # and see what it does internally ... simple create/update/delete requests or special magic ?
   module Resource
     class Base
-      def initialize(template, deploy_group)
+      def initialize(template, deploy_group, autoscaled:)
```

Comment:
any backwards compatibility concerns? Should it default to false?

Reply:
nope, I want to force every callsite to know about this